src/main/scala/viper/gobra/reporting/DefaultErrorBackTranslator.scala

```diff
@@ -71,6 +71,8 @@ object DefaultErrorBackTranslator {
         SeqIndexExceedsLengthError(node, index)
       case vprrea.SeqIndexNegative(Source(node), Source(index)) =>
         SeqIndexNegativeError(node, index)
+      case vprrea.DivisionByZero(info) =>
+        DivisionByZeroReason(Source.unapply(info))
```

Comment:
Answering https://github.com/viperproject/gobra/pull/227/files#r589612593

In the particular file that I mention in my comment (src/test/resources/regressions/features/fractional_permissions/fields/fail3.gobra), Gobra would crash with a match error if, instead of `case vprrea.DivisionByZero(info) =>`, I would have written the cases `vprrea.DivisionByZero(Source(info)) =>` and `vprrea.DivisionByZero(Syntethized(info)) =>`. I assume that it may be indeed a bug where the source information is not recorded but I did not look too much at this because it seems to be an unrelated issue.